include/private/config/locale/fr-fr.h

```diff
@@ -151,7 +150,7 @@
 
 // todo translate
```

Comment:
This todo line needs to be removed now that you've added a translation.

Reply:
all good 👍🏼